src/theme/index.hbs

```diff
@@ -78,15 +78,19 @@
                 {{> header}}
                 <div id="menu-bar" class="menu-bar">
                     <div class="left-buttons">
-                        <i id="sidebar-toggle" class="fa fa-bars" title="Toggle sidebar"></i>
-                        <i id="theme-toggle" class="fa fa-paint-brush" title="Change theme"></i>
+                        <button id="sidebar-toggle" class="icon-button" type="button" title="Toggle sidebar">
+                            <i class="fa fa-bars"></i>
+                        </button>
+                        <button id="theme-toggle" class="icon-button" type="button" title="Change theme">
+                            <i class="fa fa-paint-brush"></i>
+                        </button>
                     </div>
 
                     <h1 class="menu-title">{{ book_title }}</h1>
 
                     <div class="right-buttons">
-                        <a href="print.html">
-                            <i id="print-button" class="fa fa-print" title="Print this book"></i>
+                        <a href="print.html" title="Print this book">
```

Comment:
Should the "print" button also be a `<button>`?

Reply:
I wondered about that, too. I decided on leaving it alone because it has a valid `href`, so printing can be deep linked currently.

Reply:
Ah good point. I think it would be best to leave it as-is because `print.html` is a link to the actual print page, while the other buttons are used by javascript.